test/jdk/jdk/incubator/vector/ShortMaxVectorTests.java

```diff
@@ -1010,6 +1041,12 @@ public Object[][] shortTernaryOpProvider() {
                 toArray(Object[][]::new);
     }
 
+    @DataProvider
+    public Object[][] shortSelectFromTwoVectorOpProvider() {
+        return SHORT_GENERATOR_SELECT_FROM_TRIPLES.stream().map(List::toArray).
+                toArray(Object[][]::new);
+    }
```

Comment:
Just a control question: does this also occasionally generate examples with out-of-bounds indices? Negative out of bounds and positive out of bounds?

Reply:
Original API did throw IndexOutOfBoundsException, but later on we have moved away from exception throwing semantics to wrapping semantics.
Please find details at following comment
https://github.com/openjdk/jdk/pull/20508#issuecomment-2306344606

Reply:
VectorAPI Jtreg framework is based on testNG, our custom data providers associated with various test methods ensure to generates range of values which are beyond valid index range, this should check the wrapping scenarios.